modules/counters.js

```diff
@@ -1,3 +1,23 @@
+var QUERY_TOTAL_USERS = "select total from overall.total";
+
+var request = require('superagent');
+
 module.exports = function(app, router){
 
+  router.get('/ctl', function(req, res) {
+    request
+     .get("http://crisistextline.org/trends/data/messages.txt")
+     .end(function(response){
+        var total = response.text;
+        res.json({total: total});
+     });
+  });
+
+  router.get('/dosomething', function(req, res) {
+    app.service_data.performQuery(QUERY_TOTAL_USERS, function(rows) {
```

Comment:
I'm not sure where the numbers are coming from, but it seems to be pretty accurate:
```bash
> curl -X POST https://beta.dosomething.org/api/v1/users/get_member_count.json
{"formatted":"3,563,979","readable":"3.5 million"}
```

Not a blocker, but something to consider for the future.